lwip/lwip-2.1.1/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+# cmake-dependency-scan pkg:github/lwip-tcpip/lwip@3b53b6e481beed5787017222463626ae17943beb
```

Comment:
This added to notify the action there is a dependency that the scan misses?

Reply:
Correct, lwip is an external dependency that we don't consume via FetchContent. This annotation/comment makes sure the Action is still able to pick it up.